WooCommerce/Classes/ViewRelated/Products/Variations/Edit Attributes/AttributePickerViewController.swift

```diff
@@ -39,10 +38,16 @@ private extension AttributePickerViewController {
 
     func configureNavigationBar() {
         title = Localization.titleView
+    }
 
-        navigationItem.rightBarButtonItem = UIBarButtonItem(barButtonSystemItem: .done,
-                                                           target: self,
-                                                           action: #selector(doneButtonPressed))
+    func updateDoneButton() {
+        if viewModel.isChanged {
+            navigationItem.rightBarButtonItem = UIBarButtonItem(barButtonSystemItem: .done,
+                                                                target: self,
+                                                                action: #selector(doneButtonPressed))
+        } else {
+            navigationItem.rightBarButtonItem = nil
+        }
```

Comment:
nit: I think this would read a bit better as a `guard`
```swift
guard viewModel.isChanged else {
    navigationItem.rightBarButtonItem = nil
}

avigationItem.rightBarButtonItem = UIBarButtonItem(barButtonSystemItem: .done, target: self, action: #selector(doneButtonPressed))
```

Reply:
I've decided to keep `if-else` flow because I feel that it follows `updateDoneButton()` logic (both `true` and `false` branches are "normal" flow for "updating button" 🙂) and guard would need additional `return` line.